### ISSUE TITLE
Resource Partially Unavialable :: 102fd9bd-4737-401b-b88f-5c5b0fab94ec

### ISSUE THREAD

**Issue by fidiego** [GSA/datagov-wptheme#564]
I'm using this database to find schools by zipcode and city/state. It was working just fine last night but as of today, only states in AL-IN showed up when queried for. 

working query example. `https://inventory.data.gov/api/action/datastore_search_sql?sql=SELECT "MZIP09" as mzip09,"MSTREE09" as mstree09 ,"SCHNAM09" as schnam09,"NCESSCH" as ncessch,"MSTATE09" as mstate09 ,"MCITY09" as mcity09 FROM "102fd9bd-4737-401b-b88f-5c5b0fab94ec"%20where"MSTATE09"='IN' AND "MCITY09"='MUNSTER'`

failing query example: `https://inventory.data.gov/api/action/datastore_search_sql?sql=SELECT "MZIP09" as mzip09,"MSTREE09" as mstree09 ,"SCHNAM09" as schnam09,"NCESSCH" as ncessch,"MSTATE09" as mstate09 ,"MCITY09" as mcity09 FROM "102fd9bd-4737-401b-b88f-5c5b0fab94ec"%20where"MSTATE09"='NY' AND "MCITY09"='BROOKLYN'`

**Comment by ykhadilkar** [comment]
Looks like datastore has less number of rows than actual CSV. I am pushing CSV file again ... will update  once all the rows are pushed to datastore.

**Comment by philipashlock** [comment]
It looks like the csv is still missing many states though - it only
includes AL-IN. Was the csv replaced yesterday? Can we look at the copy
before it was replaced?

On Fri, Jan 30, 2015 at 4:09 PM, Yatin Khadilkar notifications@github.com
wrote:

> Looks like datastore has less number of rows than actual CSV. I am pushing
> CSV file again ... will update once all the rows are pushed to datastore.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/GSA/data.gov/issues/564#issuecomment-72270765.

## 

Philip Ashlock
Chief Architect, Data.gov
U.S. General Services Administration

**Comment by kvuppala** [comment]
CSV was not replaced, the api update we made triggered the datastore push.   When i downloaded the CSV it has 103K rows, and i see data upto AL state to WY state

When the datapusher pushes everything, it should work fine.

**Comment by ykhadilkar** [comment]
Ok "Successfully pushed 103959 entries". Brooklyn query should work now.

**Comment by philipashlock** [comment]
Thanks. Looks like we're good to close this. 

As a separate issue it looks like I'm having trouble downloading the whole csv. By coincidence the first time I tried downloading it, I only got the first 26mb which was only 26833 rows and seem to cut off around the state of IN. I tried again to download it and got 83.8mb which is only 84277 rows. I'm not sure if the server or client is to blame for not completing the download, but it's something we might need to look into.
